libs/cardano-ledger-repl-environment/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2024, IOHK
+All rights reserved.
```

Comment:
That's not an Apache license, as mentioned in the cabal. This should be removed, since we have one top level LICENSE file. That is the approach that is taken in this repo, for some reason